app/api/v1/endpoints/tenants.py

```
@@ -33,32 +50,102 @@ async def create_tenant(
+    return TenantResponse(
+        id=updated.id,
+        code=updated.code,
+        name=updated.name,
+        status=updated.status,
+    )
+
+
+@router.delete("/{tenant_id}", status_code=204)
+@limit_writes
+async def delete_tenant(
+    request: Request,
```

Ruff (0.15.0)

[warning] Unused function argument: `request`

(ARG001)